opensciencegrid/gwms-factory/Dockerfile

```diff
@@ -6,6 +6,8 @@ LABEL maintainer OSG Software <help@opensciencegrid.org>
 # the web-area
 RUN groupadd -o -g 996 gfactory && useradd -o -u 996 -g 996 -s /bin/sh gfactory
 
-RUN yum -y install glideinwms-factory vo-client git python3-PyYAML
+RUN    yum -y install glideinwms-factory vo-client git \
+    && pip3 install --no-cache-dir PyYAML \
```

Comment:
Can you leave a small note as to why we're using the PIP version instead of RPM?